projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,37 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: inline-block;
+    border-color: var(--tui-base-02);
+    color: var(--tui-primary);
+    border-radius: 50%;
+    width: 1rem;
+    height: 1rem;
+    outline: 0.1875rem solid var(--tui-base-01);
+    box-shadow: 0 0 0.4375rem rgba(0, 0, 0, 0.3);
+
+    &[data-mode='onDark'] {
+        outline-color: var(--tui-base-02);
```

Comment:
Let's not introduce mode, just keep it base-01. 01 is darker than in figma, but then again 02 is lighter so I think 01 is fine.

Reply:
fixed